code/procs/access.dm

```diff
@@ -87,6 +91,10 @@
 		// check implant (last, so as to avoid using it unnecessarily)
 		if (src.check_implanted_access(M))
 			return 1
+		// check for admin access override
+		if (src.admin_access_override)
+			if (M.client?.holder?.level >= LEVEL_SA)
+				return 1
```

Comment:
why not check this first

Reply:
original thinking was so it wouldn't get run all the time, but really it's probably a negligible difference.

Though having it at the top does let us skip a bunch of istypes when it's set when I think about it more

Reply:
fair enough, but it's a single var access